site/static/js/main.js

```diff
@@ -421,17 +421,9 @@ function initializeMenu() {
 }
 
 function toggleMobileMenu(icon){
-  var element = document.getElementById('navbar-mobile');
-  if(element){
-    if(element.classList.contains('visible')) {
-      if(icon) icon.classList.remove('toggled');
-      element.classList.remove('visible');
-    }
-    else
-    {
-      element.classList.add('visible');
-      if(icon) icon.classList.add('toggled');
-    }
+  var element = document.querySelector('.collapse.navbar-collapse');
+  if (element && icon) {
```

Comment:
This is nicer from the "less lines of code" POV, but is it possible to not have an icon? The previous more verbose code did cater for that situation.

Reply:
It didn't... it checked for icon in the if and the else.

Reply:
Yeah, my poor phrasing - let me rephrase. Before, it still added or removed a class on the element (and didn't matter if there was an icon), is that adding/removing class on the element no longer required (or handled elsewhere)?

Reply:
Ah right... yeah it's no longer required.  This js file was copy/pasted (I assume) from elsewhere but the template containing the elements that it selects wasn't so the `show` class is used instead of the `visible` class and seems to be handled elsewhere.